src/public/interfaces.ts

```diff
@@ -384,6 +384,16 @@ export interface Context {
    * Personal app icon y coordinate position
    */
   appIconPosition?: number;
+
+  /**
+   * Source origin from where the tab is opened
```

Comment:
is it host like teams.microsoft.com origin?

Reply:
@sachingupta Its mostly from where the tab is opened. Like say 'files' in the case of WPX tabs

Reply:
We can document what values the developer can expect here later. For now lets just keep it vague since we don't actually know what all the possible sources are yet.